### PR TITLE
ID21600: Settings fix

### PR DESCRIPTION
**Pull request by sviridovartem**

### CODE REVIEW

File: webapplication/aoi/management/commands/_ComponentExecutionHelper.py [hunk 1]
```diff
@@ -34,9 +32,15 @@ def get_environment(request:Request) -> Dict[str, str]:
             'SENTINEL2_CACHE':os.path.join(
                 settings.NOTEBOOK_POD_DATA_VOLUME_MOUNT_PATH, 
                 str(settings.SATELLITE_IMAGES_FOLDER). \
-                    replace(os.path.commonpath([settings.SATELLITE_IMAGES_FOLDER, settings.PERSISTENT_STORAGE_PATH])+'/', '')
+                    replace(os.path.commonpath([settings.SATELLITE_IMAGES_FOLDER, settings.PERSISTENT_STORAGE_PATH])+'/', ''), 
+                f"request_{request.pk}_cache"
```
Comment (adolgarev) [inline webapplication/aoi/management/commands/_ComponentExecutionHelper.py:36]: Let's add a settings for this with a backward compatible default value

Comment (sviridovartem) [reply webapplication/aoi/management/commands/_ComponentExecutionHelper.py]: fixed

**Review by adolgarev — COMMENTED** [review]
There are 2 things in 1 PR
1. Dates are optional fix - this is ok
2. Cache folder for sentinel images should be one for all containers, this is all the point of the cache - reuse images downloaded from Sentinel. If somehow errors occur these should be fixed in sentinel2tools

**Comment by sviridovartem** [comment]
> There are 2 things in 1 PR
> 
> 1. Dates are optional fix - this is ok
> 2. Cache folder for sentinel images should be one for all containers, this is all the point of the cache - reuse images downloaded from Sentinel. If somehow errors occur these should be fixed in sentinel2tools

fixed

**Review by adolgarev — APPROVED** [review]

**Pull request merged**